code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -2,10 +2,22 @@
 	name = "12g shotgun slug"
 	damage = 60
 
+/obj/projectile/bullet/shotgun_slug/executioner
+	name = "executioner slug"
+	sharpness = TRUE
+	wound_bonus = 0
+
+/obj/projectile/bullet/shotgun_slug/pulverizer
+	name = "pulverizer slug"
+	sharpness = FALSE
+	wound_bonus = 0
+
 /obj/projectile/bullet/shotgun_beanbag
 	name = "beanbag slug"
-	damage = 5
-	stamina = 55
+	damage = 10
+	stamina = 45

```

Comment:
this is actually a really significant change for bartenders

you've made it so that they can't two hit stamina crit unarmored people with their shotgun that can only hold 2 beanbag slugs in it at a time

this will cause even more bartenders to switch to just filling their shotgun with lethals so that they can actually down unarmored people with their shotgun without needing to reload it mid-fight

also, the damage increase and the wound_bonus makes beanbag slugs even LESS effective at their intended purpose of bringing people down nonlethally, as you can now accidentally wound them while trying NOT to cause permanent damage

save this nerf for another PR, please